lib/libutils/ext/sub.mk

```diff
@@ -12,7 +12,10 @@ srcs-y += fault_mitigation.c
 srcs-y += qsort_helpers.c
 srcs-y += array.c
 srcs-y += base64.c
-
+ifeq ($(CFG_CORE_SANITIZE_UNDEFINED),y)
+srcs-y += ubsan.c
+cflags-remove-ubsan.c-y += -fsanitize=undefined
```

Comment:
It makes sense not to sanitize `ubsan.c`, but this change should be mentioned in the commit message.

Reply:
Done